<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtVtkClass</class>
 <widget class="QMainWindow" name="QtVtkClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtVtk</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QVTKWidget" name="qvtkWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1601</width>
      <height>1031</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cloud">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>300</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>点云</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>30</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_texture">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>700</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>纹理</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>30</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_shelter">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>570</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1920</width>
      <height>1028</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>遮挡</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>30</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_line">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>430</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>三角</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>30</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LeapMotion">
    <property name="geometry">
     <rect>
      <x>1690</x>
      <y>40</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Leap_Motion模式</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1650</x>
      <y>40</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Leao_Motion模式已开启&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="Cursor_Button">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>鼠标模式</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Rotate_Button">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>180</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>旋转模式</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Zoom_Button">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>230</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>缩放模式</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Pause_Button">
    <property name="geometry">
     <rect>
      <x>1790</x>
      <y>200</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停LM</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Mix_Button">
    <property name="geometry">
     <rect>
      <x>1790</x>
      <y>150</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>多模式混合</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="cloud_data"/>
    <addaction name="data_save"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="clouds">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>点云</string>
   </property>
  </action>
  <action name="trips">
   <property name="text">
    <string>三角片</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>开启</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="texture">
   <property name="text">
    <string>纹理</string>
   </property>
  </action>
  <action name="file_read">
   <property name="text">
    <string>加载文件</string>
   </property>
  </action>
  <action name="cloud_data">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>文件加载</string>
   </property>
  </action>
  <action name="data_save">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>图像保存</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButton_cloud</tabstop>
  <tabstop>pushButton_shelter</tabstop>
  <tabstop>pushButton_line</tabstop>
  <tabstop>pushButton_texture</tabstop>
  <tabstop>pushButton_LeapMotion</tabstop>
 </tabstops>
 <resources>
  <include location="QtVtk.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_cloud</sender>
   <signal>clicked()</signal>
   <receiver>QtVtkClass</receiver>
   <slot>display_clouds()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1670</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>1727</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_line</sender>
   <signal>clicked()</signal>
   <receiver>QtVtkClass</receiver>
   <slot>display_lines()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1670</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>1729</x>
     <y>538</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_shelter</sender>
   <signal>clicked()</signal>
   <receiver>QtVtkClass</receiver>
   <slot>display_shelter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1670</x>
     <y>645</y>
    </hint>
    <hint type="destinationlabel">
     <x>1706</x>
     <y>676</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_texture</sender>
   <signal>clicked()</signal>
   <receiver>QtVtkClass</receiver>
   <slot>display_texture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1740</x>
     <y>783</y>
    </hint>
    <hint type="destinationlabel">
     <x>1703</x>
     <y>812</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_LeapMotion</sender>
   <signal>clicked()</signal>
   <receiver>QtVtkClass</receiver>
   <slot>LM()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1760</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>1813</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Cursor_Button</sender>
   <signal>clicked()</signal>
   <receiver>QtVtkClass</receiver>
   <slot>Pattern_Select()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1734</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>1665</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Rotate_Button</sender>
   <signal>clicked()</signal>
   <receiver>QtVtkClass</receiver>
   <slot>Pattern_Select()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1684</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>1661</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Zoom_Button</sender>
   <signal>clicked()</signal>
   <receiver>QtVtkClass</receiver>
   <slot>Pattern_Select()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1688</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>1662</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Pause_Button</sender>
   <signal>clicked()</signal>
   <receiver>QtVtkClass</receiver>
   <slot>Pattern_Select()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1810</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>1669</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Mix_Button</sender>
   <signal>clicked()</signal>
   <receiver>QtVtkClass</receiver>
   <slot>Pattern_Select()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1847</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>1887</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menuBar</sender>
   <signal>triggered(QAction*)</signal>
   <receiver>QtVtkClass</receiver>
   <slot>Menu_Action(QAction*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>11</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>12</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>display_clouds()</slot>
  <slot>display_lines()</slot>
  <slot>display_shelter()</slot>
  <slot>display_texture()</slot>
  <slot>LM()</slot>
  <slot>Pattern_Select()</slot>
  <slot>Menu_Action(QAction*)</slot>
  <slot>Menu_Action()</slot>
 </slots>
</ui>
